common/djangoapps/django_sudo_helpers/templates/sudo/third_party_auth_links.html

```diff
@@ -0,0 +1,25 @@
+{% load i18n third_party_auth_links %}
+<script type="text/javascript">
+    function thirdPartySignin(event, url) {
+      event.preventDefault();
+      window.location.href = url;
+    }
+</script>
+<span class="deco-divider">
```

Comment:
Move this decorator inside the if enabled block (line 16) so it doesn't show if providers are not enabled.